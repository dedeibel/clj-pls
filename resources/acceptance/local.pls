[Playlist]
File1=Linkin Park - Minutes To Midnight/01 Wake.ogg
Title1=Wake
Length1=100
File2=Linkin Park - Minutes To Midnight/02 Given Up.ogg
Title2=Given Up
Length2=189
File3=Linkin Park - Minutes To Midnight/03 Leave Out All The Rest.ogg
Title3=Leave Out All The Rest
Length3=209
File4=Linkin Park - Minutes To Midnight/04 Bleed It Out.ogg
Title4=Bleed It Out
Length4=164
File5=Linkin Park - Minutes To Midnight/05 Shadow Of The Day.ogg
Title5=Shadow Of The Day
Length5=289
File6=Linkin Park - Minutes To Midnight/06 What I've Done.ogg
Title6=What I've Done
Length6=205
File7=Linkin Park - Minutes To Midnight/07 Hands Held High.ogg
Title7=Hands Held High
Length7=233
File8=Linkin Park - Minutes To Midnight/08 No More Sorrow.ogg
Title8=No More Sorrow
Length8=221
File9=Linkin Park - Minutes To Midnight/09 Valentine's Day.ogg
Title9=Valentine's Day
Length9=196
File10=Linkin Park - Minutes To Midnight/10 In Between.ogg
Title10=In Between
Length10=196
File11=Linkin Park - Minutes To Midnight/11 In Pieces.ogg
Title11=In Pieces
Length11=218
File12=Linkin Park - Minutes To Midnight/12 The Little Things Give You Away.ogg
Title12=The Little Things Give You Away
Length12=383
File13=Rammstein - Liebe ist für alle da/01 Rammlied.mp3
Title13=Rammlied
Length13=319
File14=Rammstein - Liebe ist für alle da/02 Ich tu dir Weh.mp3
Title14=Ich tu dir Weh
Length14=302
File15=Rammstein - Liebe ist für alle da/03 Waidmanns Heil.mp3
Title15=Waidmanns Heil
Length15=213
File16=Rammstein - Liebe ist für alle da/04 Haifisch.mp3
Title16=Haifisch
Length16=225
File17=Rammstein - Liebe ist für alle da/05 B________.mp3
Title17=B********
Length17=254
File18=Rammstein - Liebe ist für alle da/06 Frühling in Paris.mp3
Title18=Frühling in Paris
Length18=285
File19=Rammstein - Liebe ist für alle da/07 Wiener Blut.mp3
Title19=Wiener Blut
Length19=232
File20=Rammstein - Liebe ist für alle da/08 Pussy.mp3
Title20=Pussy
Length20=239
File21=Rammstein - Liebe ist für alle da/09 Liebe ist für alle da.mp3
Title21=Liebe ist für alle da
Length21=205
File22=Rammstein - Liebe ist für alle da/10 Mehr.mp3
Title22=Mehr
Length22=248
File23=Rammstein - Liebe ist für alle da/11 Roter Sand.mp3
Title23=Roter Sand
Length23=239
File24=Rammstein - Liebe ist für alle da/12 Führe mich.mp3
Title24=Führe mich
Length24=273
File25=Rammstein - Liebe ist für alle da/13 Donaukinder.mp3
Title25=Donaukinder
Length25=318
File26=Rammstein - Liebe ist für alle da/14 Halt.mp3
Title26=Halt
Length26=261
File27=Rammstein - Liebe ist für alle da/15 Roter Sand (Orchester Version).mp3
Title27=Roter Sand (Orchester Version)
Length27=245
File28=Rammstein - Liebe ist für alle da/16 Liese.mp3
Title28=Liese
Length28=236
File29=Rammstein - Liebe ist für alle da/17 Rammlied (Thrash-Terpiece Remix).mp3
Title29=Rammlied (Thrash-Terpiece Remix)
Length29=380
NumberOfEntries=29
Version=2
