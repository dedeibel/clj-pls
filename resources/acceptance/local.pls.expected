 {
:entries 29,
:version "2",
:files (
  {:length 100,
  :title "Wake",
  :url "Linkin Park - Minutes To Midnight/01 Wake.ogg"}
  {:length 189,
  :title "Given Up",
  :url "Linkin Park - Minutes To Midnight/02 Given Up.ogg"}
  {:title "Leave Out All The Rest",
  :length 209,
  :url "Linkin Park - Minutes To Midnight/03 Leave Out All The Rest.ogg"}
  {:title "Bleed It Out",
  :url "Linkin Park - Minutes To Midnight/04 Bleed It Out.ogg",
  :length 164}
  {:length 289,
  :title "Shadow Of The Day",
  :url "Linkin Park - Minutes To Midnight/05 Shadow Of The Day.ogg"}
  {:url "Linkin Park - Minutes To Midnight/06 What I've Done.ogg",
  :length 205,
  :title "What I've Done"}
  {:title "Hands Held High",
  :url "Linkin Park - Minutes To Midnight/07 Hands Held High.ogg",
  :length 233}
  {:title "No More Sorrow",
  :url "Linkin Park - Minutes To Midnight/08 No More Sorrow.ogg",
  :length 221}
  {:url "Linkin Park - Minutes To Midnight/09 Valentine's Day.ogg",
  :title "Valentine's Day",
  :length 196}
  {:title "In Between",
  :url "Linkin Park - Minutes To Midnight/10 In Between.ogg",
  :length 196}
  {:url "Linkin Park - Minutes To Midnight/11 In Pieces.ogg",
  :length 218,
  :title "In Pieces"}
  {:title "The Little Things Give You Away",
  :length 383,
  :url "Linkin Park - Minutes To Midnight/12 The Little Things Give You Away.ogg"}
  {:title "Rammlied",
  :url "Rammstein - Liebe ist für alle da/01 Rammlied.mp3",
  :length 319}
  {:url "Rammstein - Liebe ist für alle da/02 Ich tu dir Weh.mp3",
  :title "Ich tu dir Weh",
  :length 302}
  {:length 213,
  :url "Rammstein - Liebe ist für alle da/03 Waidmanns Heil.mp3",
  :title "Waidmanns Heil"}
  {:url "Rammstein - Liebe ist für alle da/04 Haifisch.mp3",
  :length 225,
  :title "Haifisch"}
  {:title "B********",
  :length 254,
  :url "Rammstein - Liebe ist für alle da/05 B________.mp3"}
  {:title "Frühling in Paris",
  :length 285,
  :url "Rammstein - Liebe ist für alle da/06 Frühling in Paris.mp3"}
  {:length 232,
  :title "Wiener Blut",
  :url "Rammstein - Liebe ist für alle da/07 Wiener Blut.mp3"}
  {:title "Pussy",
  :length 239,
  :url "Rammstein - Liebe ist für alle da/08 Pussy.mp3"}
  {:length 205,
  :title "Liebe ist für alle da",
  :url "Rammstein - Liebe ist für alle da/09 Liebe ist für alle da.mp3"}
  {:url "Rammstein - Liebe ist für alle da/10 Mehr.mp3",
  :length 248,
  :title "Mehr"}
  {:length 239,
  :url "Rammstein - Liebe ist für alle da/11 Roter Sand.mp3",
  :title "Roter Sand"}
  {:title "Führe mich",
  :url "Rammstein - Liebe ist für alle da/12 Führe mich.mp3",
  :length 273}
  {:title "Donaukinder",
  :url "Rammstein - Liebe ist für alle da/13 Donaukinder.mp3",
  :length 318}
  {:url "Rammstein - Liebe ist für alle da/14 Halt.mp3",
  :length 261,
  :title "Halt"}
  {:length 245,
  :title "Roter Sand (Orchester Version)",
  :url "Rammstein - Liebe ist für alle da/15 Roter Sand (Orchester Version).mp3"}
  {:url "Rammstein - Liebe ist für alle da/16 Liese.mp3",
  :length 236,
  :title "Liese"}
  {:url "Rammstein - Liebe ist für alle da/17 Rammlied (Thrash-Terpiece Remix).mp3",
  :title "Rammlied (Thrash-Terpiece Remix)",
  :length 380}
)}
